mr/src/main/java/org/apache/iceberg/mr/hive/serde/objectinspector/IcebergRecordObjectInspector.java

```diff
@@ -72,11 +72,17 @@ public StructField getStructFieldRef(String name) {
 
   @Override
   public Object getStructFieldData(Object o, StructField structField) {
+    if (o == null) {
+      return null;
+    }
```

Comment:
Nit: Please add a newline between control flow blocks (so after this if block).

Same comment for below 👍